lib/osf-components/addon/components/institution-carousel/template.hbs

```diff
@@ -6,8 +6,6 @@
         @showIndicators={{false}}
         @prevControlLabel={{t 'general.previous'}}
         @nextControlLabel={{t 'general.next'}}
-        @prevControlIcon='fa fa-angle-left'
-        @nextControlIcon='fa fa-angle-right'
```

Comment:
Hey, there are default controls we can use. Let's use those instead of breaking this.